docs/resources/dms_rocketmq_topic.md

```diff
@@ -0,0 +1,75 @@
+---
+subcategory: "Distributed Message Service (DMS)"
+---
+
+# huaweicloud_dms_rocketmq_topic
+
+Manages DMS RocketMQ topic resource within HuaweiCloud.
+
+## Example Usage
+
+```HCL
+variable "instance_id" {}
+
+resource "huaweicloud_dms_rocketmq_topic" "test" {
+  instance_id    = var.instance_id
+  name           = "topic_test"
+  brokers        = ["broker-0","broker-1"]
+  queue_num      = 3
+  permission     = "all"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String, ForceNew) Specifies the region in which to create the resource.
+  If omitted, the provider-level region will be used. Changing this parameter will create a new resource.
+
+* `instance_id` - (Required, String, ForceNew) Specifies the ID of the rocketMQ instance.
+  Changing this parameter will create a new resource.
+
+* `name` - (Required, String, ForceNew) Specifies the name of the topic.
+  Changing this parameter will create a new resource.
+
+* `brokers` - (Required, List, ForceNew) Specifies the list of associated brokers of the topic.
+  Changing this parameter will create a new resource.
+  The [BrokerRef](#DmsRocketMQTopic_BrokerRef) structure is documented below.
+
+* `queue_num` - (Optional, Int, ForceNew) Specifies the number of queues. Default to 3.
+  Changing this parameter will create a new resource.
+
+* `permission` - (Optional, String) Specifies the permissions of the topic.
+  Value options: **all**, **sub**, **pub**. Default to all.
+
+* `read_queue_num` - (Optional, Int) Specifies the total number of read queues.
+
+* `write_queue_num` - (Optional, Int) Specifies the total number of write queues.
+
+<a name="DmsRocketMQTopic_BrokerRef"></a>
+The `BrokerRef` block supports:
+
+* `broker_name` - (Optional, String) Indicates the name of the broker.
+
+* `read_queue_num` - (Optional, Int) Indicates the read queues number of the broker. It's useless when create a topic.
+
+* `write_queue_num` - (Optional, Int) Indicates the read queues number of the broker. It's useless when create a topic.
+
+## Attributes Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+* `id` - The resource ID.
+
+* `read_queue_num` - Specifies the total number of read queues.
+
+* `write_queue_num` - Specifies the total number of write queues.

```

Comment:
remove these attributes, as they are already contained in the Arguments

Reply:
done

Reply:
they have been changed to total_read_queue_num and total_write_queue_num